Insert Note on VMs and kiwi-ng boxbuild (#106)
rockstor/rockstor-installer: README.md

Assigning too much ram will crash your host, so be sure to set a safe amount smaller than your current available host ram.
Arguments before the `--` are passed to boxbuild, and after are passed to the kiwi-ng build itself.

**Note on using a host OS within a Virtual Machine:**

> If using a Linux OS on a Virtual Machine, it seems that any shared folders to the machine hosting the VM, are not properly recognized by the KVM/QEMU that is generated by `kiwi-ng`'s boxbuild approach. This can lead to `chroot` errors and terminated installer generation. The recommendation in this case to clone the installer git directly into the VM directory and edit the relevant files (e.g. `rockstor.kiwi`) before executing the box build (or edit them outside of the VM and transfer them back into the VM's installer directory using a shared folder. At the end of installer creation the `.iso` file has to be then copied from the VM directory back onto the VM's host for further use.

Reviewer: ```suggestion
If using a Linux OS in a Virtual Machine, any shared folders from the host are not properly recognized by the KVM/QEMU that is generated by `kiwi-ng`'s boxbuild approach.
This can lead to `chroot` errors and terminate the installer generation.
The recommendation in this case is to clone the installer git directly into the VM directory and edit the relevant files (e.g. `rockstor.kiwi`) before executing the box build; or edit them outside of the VM and transfer them back into the VM's installer directory using a shared folder.
At the end of installer creation the `.iso` file must then be copied from within the VM directory back onto the VM's host for further use.
```

Reviewer: @Hooverdan96 Nice. I've made some minor re-wording suggestions and moved to semantic line feeds (https://rhodesmill.org/brandon/2012/one-sentence-per-line/). One sentence or major part there-of per line.
If you are happy with these suggestions go ahead and commit them and I think we can pop this one in.
